packages/checkbox/src/use-checkbox.ts

```diff
@@ -106,7 +113,7 @@ export function useCheckbox(props: UseCheckboxProps = {}) {
 
   const ref = useRef<HTMLInputElement>(null)
 
-  const [checkedState, setCheckedState] = useState(!!defaultIsChecked)
+  const [checkedState, setCheckedState] = useState(!!defaultChecked)
```

Comment:
Would it be better to remind users ?
```suggestion
 if(props.hasOwnproperty('defaultIsChecked')) {
   console.warn('defaultIsChecked is deprecated! Please use the `defaultChecked` prop, which mirrors default')
 }
  const [checkedState, setCheckedState] = useState(!!defaultChecked)
```

Reply:
Great idea! Added a warning to both `useCheckbox` and `useRadio` using our `warn` utility: https://github.com/chakra-ui/chakra-ui/pull/2876/files#diff-445514188dc73f74a48ca922881ed7816acdcf8acca6e0bf2fb39c2e4ab340bfR124-R130